<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipCalculator</class>
 <widget class="QWidget" name="PipCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forex Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>377</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculators">
    <property name="title">
     <string>&amp;Calculators</string>
    </property>
    <addaction name="actionPip_calculator"/>
    <addaction name="actionMargin_calculator"/>
    <addaction name="actionLot_Size_Calculator"/>
    <addaction name="actionProfit_and_Loss_Calculator"/>
    <addaction name="actionPosition_Size_Calculator"/>
    <addaction name="actionRisk_Reward_Calculator"/>
    <addaction name="actionSwap_Calculator"/>
    <addaction name="actionCurrency_Converter"/>
    <addaction name="actionFibonacci_Calculator"/>
    <addaction name="actionVolatility_Calculator"/>
    <addaction name="actionBreak_Even_Calculator"/>
    <addaction name="actionCompound_Interest_Calculator"/>
    <addaction name="actionRisk_Management_Calculator"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="title">
     <string>&amp;Languages</string>
    </property>
    <widget class="QMenu" name="menuAsian_2">
     <property name="title">
      <string>Asian</string>
     </property>
     <addaction name="actionHindi_2"/>
     <addaction name="actionBengali_2"/>
     <addaction name="actionJapanese_2"/>
     <addaction name="actionKorean_2"/>
     <addaction name="actionVietnamese_2"/>
     <addaction name="actionThai_2"/>
    </widget>
    <widget class="QMenu" name="menuEuropean">
     <property name="title">
      <string>European</string>
     </property>
     <addaction name="actionFrench_2"/>
     <addaction name="actionGerman_2"/>
     <addaction name="actionRussian_2"/>
     <addaction name="actionPortuguese_2"/>
     <addaction name="actionItalian"/>
    </widget>
    <widget class="QMenu" name="menuMiddle_Eastern">
     <property name="title">
      <string>Middle Eastern</string>
     </property>
     <addaction name="actionPersian_Farsi"/>
     <addaction name="actionTurkish"/>
     <addaction name="actionHebrew"/>
     <addaction name="actionPashto"/>
    </widget>
    <widget class="QMenu" name="menuAfrican">
     <property name="title">
      <string>African</string>
     </property>
     <addaction name="actionSwahili"/>
     <addaction name="actionHausa"/>
     <addaction name="actionAmharic"/>
     <addaction name="actionYoruba"/>
     <addaction name="actionZulu"/>
    </widget>
    <widget class="QMenu" name="menuIndigenous">
     <property name="title">
      <string>Indigenous</string>
     </property>
     <addaction name="actionHaitian_Creole"/>
     <addaction name="actionQuechua"/>
     <addaction name="actionGuarani"/>
     <addaction name="actionNahuatl"/>
    </widget>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
    <addaction name="actionChinese_2"/>
    <addaction name="actionArabic"/>
    <addaction name="separator"/>
    <addaction name="menuAsian_2"/>
    <addaction name="menuEuropean"/>
    <addaction name="menuMiddle_Eastern"/>
    <addaction name="menuAfrican"/>
    <addaction name="menuIndigenous"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="aboutQt"/>
    <addaction name="aboutForexCalculator"/>
   </widget>
   <addaction name="menuCalculators"/>
   <addaction name="menuLanguages"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>31</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Base Currency</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Quote Currency</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lotSizeInput">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>124</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>119</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lot Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="exchangeRateInput">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>174</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>optional</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>169</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Exchange Rate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>27</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pip Size</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>224</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="baseCurrencyCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="quoteCurrencyCombo">
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="calculateButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pip Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pipValueResult">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>320</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <action name="aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="aboutForexCalculator">
   <property name="text">
    <string>About Forex Calculator</string>
   </property>
  </action>
  <action name="actionPip_calculator">
   <property name="text">
    <string>Pip Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionMargin_calculator">
   <property name="text">
    <string>Margin Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLot_Size_Calculator">
   <property name="text">
    <string>Lot Size Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionProfit_and_Loss_Calculator">
   <property name="text">
    <string>Profit and Loss Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPosition_Size_Calculator">
   <property name="text">
    <string>Position Size Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRisk_Reward_Calculator">
   <property name="text">
    <string>Risk/Reward Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSwap_Calculator">
   <property name="text">
    <string>Swap Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCurrency_Converter">
   <property name="text">
    <string>Currency Converter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionFibonacci_Calculator">
   <property name="text">
    <string>Fibonacci Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionVolatility_Calculator">
   <property name="text">
    <string>Volatility Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionBreak_Even_Calculator">
   <property name="text">
    <string>Break-Even Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionCompound_Interest_Calculator">
   <property name="text">
    <string>Compound Interest Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionChinese_2">
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionHindi">
   <property name="text">
    <string>Hindi</string>
   </property>
  </action>
  <action name="actionBengali">
   <property name="text">
    <string>Bengali</string>
   </property>
  </action>
  <action name="actionPunjabi">
   <property name="text">
    <string>Punjabi</string>
   </property>
  </action>
  <action name="actionUrdu">
   <property name="text">
    <string>Urdu</string>
   </property>
  </action>
  <action name="actionVietnamese">
   <property name="text">
    <string>Vietnamese</string>
   </property>
  </action>
  <action name="actionIndonesian_Bahasa_Indonesia">
   <property name="text">
    <string>Indonesian (Bahasa Indonesia)</string>
   </property>
  </action>
  <action name="actionTagalog_Filipino">
   <property name="text">
    <string>Tagalog (Filipino)</string>
   </property>
  </action>
  <action name="actionKorean">
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="actionTamil">
   <property name="text">
    <string>Tamil</string>
   </property>
  </action>
  <action name="actionThai">
   <property name="text">
    <string>Thai</string>
   </property>
  </action>
  <action name="actionBurmese">
   <property name="text">
    <string>Burmese</string>
   </property>
  </action>
  <action name="actionKhmer">
   <property name="text">
    <string>Khmer</string>
   </property>
  </action>
  <action name="actionNepali">
   <property name="text">
    <string>Nepali</string>
   </property>
  </action>
  <action name="actionHindi_2">
   <property name="text">
    <string>Hindi</string>
   </property>
  </action>
  <action name="actionBengali_2">
   <property name="text">
    <string>Bengali</string>
   </property>
  </action>
  <action name="actionJapanese_2">
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionKorean_2">
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="actionVietnamese_2">
   <property name="text">
    <string>Vietnamese</string>
   </property>
  </action>
  <action name="actionThai_2">
   <property name="text">
    <string>Thai</string>
   </property>
  </action>
  <action name="actionFrench_2">
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionGerman_2">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionRussian_2">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionPortuguese_2">
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionItalian">
   <property name="text">
    <string>Italian</string>
   </property>
  </action>
  <action name="actionPersian_Farsi">
   <property name="text">
    <string>Persian (Farsi)</string>
   </property>
  </action>
  <action name="actionTurkish">
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
  <action name="actionHebrew">
   <property name="text">
    <string>Hebrew</string>
   </property>
  </action>
  <action name="actionPashto">
   <property name="text">
    <string>Pashto</string>
   </property>
  </action>
  <action name="actionSwahili">
   <property name="text">
    <string>Swahili</string>
   </property>
  </action>
  <action name="actionAmharic">
   <property name="text">
    <string>Amharic</string>
   </property>
  </action>
  <action name="actionHausa">
   <property name="text">
    <string>Hausa</string>
   </property>
  </action>
  <action name="actionYoruba">
   <property name="text">
    <string>Yoruba</string>
   </property>
  </action>
  <action name="actionZulu">
   <property name="text">
    <string>Zulu</string>
   </property>
  </action>
  <action name="actionHaitian_Creole">
   <property name="text">
    <string>Haitian Creole</string>
   </property>
  </action>
  <action name="actionQuechua">
   <property name="text">
    <string>Quechua</string>
   </property>
  </action>
  <action name="actionGuarani">
   <property name="text">
    <string>Guarani</string>
   </property>
  </action>
  <action name="actionNahuatl">
   <property name="text">
    <string>Nahuatl</string>
   </property>
  </action>
  <action name="actionRisk_Management_Calculator">
   <property name="text">
    <string>Risk Management Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
